src/Platform.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Utopia\Platform;
+
+use Utopia\App;
+
+abstract class Platform {
+    protected array $services = [];
+
+    public function init():void
+    {
+        foreach ($this->services as $key => $service) {
+            if($service->getType() === Service::TYPE_HTTP) {
+                foreach ($service->getActions() as $key => $action) {
+                    /** @var Action $action */
+                    switch($action->getHttpMethod()) {
+                        // should I make App::addRoute(method, path) public, it's protected now
+                        case 'post':
```

Comment:
I think we can skip the per method approach and just use `addRoute`: https://github.com/utopia-php/framework/blob/584bab40e24aa58fa3884ca38f9485c2404196c9/src/App.php#L470

Reply:
You can also just regularly init the `new Route` class

Reply:
regularly initing new Route then I'll need a new method to add route to the App::$routes

Reply:
The method I linked to should help, no?